docs/getting-started/server/advanced-setup.md

```diff
@@ -46,8 +46,9 @@ subscription.
 
 ## Emails
 
-Docker compose will spin up a local smtp server that can be used. Go to `http://localhost:1080` to
-view the email content locally.
+Docker compose will spin up a local smtp server, Mailcatcher, that can be used. See the
+[Setup Guide](https://contributing.bitwarden.com/getting-started/server/guide/#mailcatcher) for more
```

Comment:
Please use relative links when linking within the docs. e.g. `[Setup Guide](./guide.md#mailcatcher)`.